spinel/stream.py

```diff
@@ -50,9 +50,9 @@ def close(self):
 class StreamSerial(IStream):
     """ An IStream interface implementation for serial devices. """
 
-    def __init__(self, dev, baudrate=115200):
+    def __init__(self, dev, baudrate=115200, rtscts=False):
         try:
-            self.serial = serial.Serial(dev, baudrate)
+            self.serial = serial.Serial(port=dev, baudrate=baudrate, rtscts=rtscts)
```

Comment:
nit: I think we should use the assignments in parameter list when we use ambiguous values like "115200", "True" etc. I think it can be omitted here.

Reply:
Discussed F2F, leaving as is. The reason is the fact that `rtscts` is not the third argument in the Serial constructor function.